cmd/multy-back/main.go

```diff
@@ -17,12 +17,14 @@ import (
 )
 
 var (
-	log = slf.WithContext("main")
+	log = slf.WithContext("multy-back")
 
+	// Set externaly during build
 	branch    string
 	commit    string
-	buildtime string
 	lasttag   string
+	buildtime string
+
```

Comment:
If it is interesting to aim for reproducible builds then "Timestamps are best avoided"

> If a date is required to give users an idea on when the software was made, it is better to use a date that is relevant to the source code instead of the build: old software can always be built later. Like version information, it’s best to extract such a date from the revision control system or from a changelog.

https://reproducible-builds.org/docs/timestamps/